src/commonTest/kotlin/software/momento/kotlin/sdk/auth/CredentialProviderTest.kt

```diff
@@ -14,9 +17,11 @@ class CredentialProviderTest : UsingTestRunner() {
         private const val CACHE_ENDPOINT_LEGACY = "cache.example.com"
         private const val CONTROL_ENDPOINT_V1 = "control.test.momentohq.com"
         private const val CACHE_ENDPOINT_V1 = "cache.test.momentohq.com"
-
+        private const val TEST_ENDPOINT = "testEndpoint"
+        private const val TEST_ENDPOINT_ENV_VAR = "testEndpoint"
         // Test tokens are all fake and nonfunctional.
-
+        private const val TEST_ENV_VAR = "MOMENTO_TEST_V2_ENV_VAR"
+	    private const val TEST_V2_API_KEY      = "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJ0IjoiZyJ9.LloWc3qLRkBm_djlOjXE8wNSENqOay17xHLJR5XIr0cwkyhhh8w_oBaiQDktBkOvh-wKLQGUKavSQuOwXEb2_g"
```

Comment:
I made this fake v2 api key with the claim `"t": "g"` embedded:
```suggestion
	    private const val TEST_V2_API_KEY      = "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJ0IjoiZyIsImp0aSI6InNvbWUtaWQifQ.GMr9nA6HE0ttB6llXct_2Sg5-fOKGFbJCdACZFgNbN1fhT6OPg_hVc8ThGzBrWC_RlsBpLA1nzqK3SOJDXYxAw"
```